.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - '0.10'
+  - '4.2.2'
```

Comment:
@kevinold I can change the `node version` back to `0.10` or `0.12` but jest v0.5+  will not work with node 0.12. See https://github.com/facebook/jest/issues/427#issuecomment-149773152

Right now, you can still run/build (grunt) `griddle` on `node v0.10.40` with this PR, but tests will fail.